README.md

```diff
@@ -100,59 +107,57 @@ somewhere local:
 $ sonobuoy retrieve .
 ```
 
-This copies a single `.tar.gz` snapshot from the Sonobuoy pod into your local
+You can then extra the resulting `.tar.gz` snapshot from the Sonobuoy pod into your local
 `.` directory. Extract the contents into `./results` with:
 
 ```
+# If you only have one tar.gz in the pwd
 mkdir ./results; tar xzf *.tar.gz -C ./results
+
+# Sonobuoy retrieve echos the created file to stdout so you can capture and manipulate it.
+$ output=$(sonoobuoy retrieve)
+$ mkdir ./results; tar xzf $output -C ./results
```

Comment:
Side note: I wish github allowed multi-language snippets. I always feel bad listing directions for Mac/Linux without having a Windows version.